examples/erc165/index.mjs

```diff
@@ -0,0 +1,40 @@
+import { loadStdlib } from "@reach-sh/stdlib";
```

Comment:
This is fine for 2261, but 224[789] should use ethers directly like the other tests